datadog_checks_base/datadog_checks/base/data/agent_requirements.in

```diff
@@ -96,7 +95,7 @@ requests==2.27.1; python_version < '3.0'
 requests==2.28.1; python_version > '3.0'
 rethinkdb==2.4.9
 scandir==1.10.0
-securesystemslib[crypto,pynacl]==0.20.1
+securesystemslib[crypto,pynacl]==0.25.0; python_version > '3.0'
```

Comment:
`datadog_checks_dev` still uses `0.20.1` of `securesystemslib`: https://github.com/DataDog/integrations-core/blob/03c3327054097750abcbc33dd7950f20a2982dc2/datadog_checks_dev/pyproject.toml#L76 for [signing release PRs](https://github.com/DataDog/integrations-core/blob/03c3327054097750abcbc33dd7950f20a2982dc2/datadog_checks_dev/datadog_checks/dev/tooling/signing.py#L10). Maybe we should keep `0.20.1` for `python_version < '3.0'`?

Reply:
Those are CLI dependencies, which are not used by any python 2 code, and which are not shipped with the agent.